client/dashboard/app/context.tsx

```diff
@@ -45,6 +45,7 @@ export type AppConfig = {
 		default: OnboardingLink;
 		withAI: OnboardingLink;
 	};
+	optIn?: boolean;
```

Comment:
Like I said on Slack, I think we should force people to say `true` or `false`. p1760929164208319/1760928584.607169-slack-CRWCHQGUB